evm-contracts/src/v0.7/dev/OperatorForwarder.sol

```diff
@@ -4,23 +4,70 @@ pragma solidity ^0.7.0;
 import "../interfaces/OperatorInterface.sol";
 import "./ConfirmedOwner.sol";
 
-contract OperatorForwarder {
-  address public immutable authorizedSender1;
-  address public immutable authorizedSender2;
-  address public immutable authorizedSender3;
+contract OperatorForwarder is ConfirmedOwner{
+
+  mapping(address => bool) private s_authorizedSenders;
+  address[] private s_authorizedSenderList;
 
   address public immutable linkAddr;
 
+  event AuthorizedSendersChanged(
+    address[] senders
+  );
+
   constructor(
     address link
-  ) {
+  ) 
+    ConfirmedOwner(msg.sender)
```

Comment:
Does this set msg.sender as `s_owner` on ConfirmedOwner?

Reply:
Correct! This calls the `ConfirmedOwner` constructor, which accepts an address and sets it as the owner.